app/views/posts/main_page/_important_news.html.slim

```diff
@@ -0,0 +1,8 @@
+h3 Important
+table
+  = render 'posts/main_page/table_head', post: @post
+  tbody
+    - @posts.each do |post|
+      - unless post.visibility_id == 5
+        - if post.importance == true
```

Comment:
Ruby is a powerful language. ActiveRecord gives you a boolean method, called `Post#importance?`

Reply:
Secondly, you should never do `... == true` in Ruby.